### PR TITLE
flesh out README

### PR DESCRIPTION
**Pull request by Ravna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
-# ppa
-A PPA repository for SecureDNA packages
+# PPA
+A PPA repository for SecureDNA packages.  Currently supports [Ubuntu amd64](https://github.com/SecureDNA/ppa).
```
Comment (swooster) [inline README.md]: It feels a little odd to have have this link back to itself (I figure this URL is the most likely way for somebody to stumble across this readme).

Comment (Ravna) [reply README.md]: Uh, you're right. :)

File: README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+
+SecureDNA is a free, non-profit screening platform designed to safeguard DNA synthesis everywhere.
+
+- You can find information about the project as a whole, including papers and a live demo, at the [main site](https://securedna.org).
+- Source for the client and servers is available in our [monorepo](https://github.com/SecureDNA/SecureDNA).
+- To use the production servers with real databases, you'll need to [obtain certificates](https://securedna.org/start/), but you can compile the entire system below and run test servers with test databases and your own certificates.
```
Comment (swooster) [inline README.md]: I don't think of the PPA as involving compilation. Also, does it include the servers?

Comment (Ravna) [reply README.md]: I should maybe read what I copy from other repos. :)  Now I've rewritten a bunch of it for clarity; tnx.

**Review by swooster — APPROVED** [review]

**Pull request merged**